src/components/TextSize.js

```diff
@@ -1,36 +1,59 @@
 import { h } from 'preact'
 import { useContext } from 'preact/hooks'
-import { useI18n, useSoftkey, useArticleTextSize } from 'hooks'
+import { useI18n, useSoftkey } from 'hooks'
 import { articleTextSize } from 'utils'
-import { DirectionContext } from 'contexts'
+import { FontContext, DirectionContext } from 'contexts'
 
-export const TextSize = ({ close }) => {
+export const TextSize = ({ close, closeAll }) => {
   const i18n = useI18n()
   const { dirState } = useContext(DirectionContext)
-  const [textSize, setTextSize] = useArticleTextSize()
+  const { textSize, setTextSize } = useContext(FontContext)
+  const originalTextSize = articleTextSize.get()
```

Comment:
This will run every time so there is no guarantee that it will contain the true original value.

To initialize it on the first run and keep it for the lifecycle of the component you can do:

`const [originalTextSize] = useState(articleTextSize.get())`